File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1132.c
#include <stdio.h>

int main(){
    int x, y;
    scanf("%d %d", &x, &y);
    if(x>y){
        int temp=x;
        x=y;
        y=temp;
    }
    int soma=0;
    for(x;x<=y;x++){
        if(x%13!=0){
            soma+=x;
        }
    }
    printf("%d\n", soma);

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1132.lua
x = io.read("*number")
y = io.read("*number")

if(x>y) then
	temp=x;
	x=y;
	y=temp;
end
soma=0;
for i=x,y,1 do
	if(i%13~=0) then
		soma=soma+i;
	end
end
print(soma);
